enos/enos-modules.hcl

```diff
@@ -22,8 +22,10 @@ module "boundary" {
     "Environment" : var.environment
   }
 
-  ssh_aws_keypair       = var.aws_ssh_keypair_name
```

Comment:
Was this moved down to be sorted alphabetically? If so, then `boundary_binary_name` and `boundary_license` should be swapped?

Reply:
spelling is hard!